src/jit/codegenarmarch.cpp

```diff
@@ -343,7 +343,11 @@ void CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
         case GT_LABEL:
             genPendingCallLabel       = genCreateTempLabel();
             treeNode->gtLabel.gtLabBB = genPendingCallLabel;
+#if defined(_TARGET_ARM)
```

Comment:
`_TARGET_ARM_` as well as `_TARGET_ARM64_` should both have an ending underscore.